python/chigger/exodus/ExodusReader.py

```diff
@@ -158,49 +171,50 @@ def update(self, **kwargs):
         self.__initializeTimeInformation()
         self.__current = self.__getTimeInformation()
         self.__vtkreader.SetFileName(None) # http://vtk.1045678.n5.nabble.com/How-to-re-load-time-information-in-ExodusIIReader-tp5741615.html pylint: disable=line-too-long
-        self.__vtkreader.SetFileName(self.__current.filename)
-        self.__vtkreader.SetTimeStep(self.__current.index)
-        self.__vtkreader.UpdateInformation()
-        self.__vtkreader.Modified()
-
-        # Displacement Settings
-        if self.getOption('displacements'):
-            self.__vtkreader.ApplyDisplacementsOn()
-            self.__vtkreader.SetDisplacementMagnitude(self.getOption('displacement_magnitude'))
-        else:
-            self.__vtkreader.ApplyDisplacementsOff()
+        with lockFile(self.__current.filename):
```

Comment:
The diff doesn't really show it but this was the only thing  added. The rest was just an indentation change.